event-asynchronous/src/main/java/com/iluwatar/event/asynchronous/App.java

```diff
@@ -0,0 +1,206 @@
+ * of the complex issues inherent in multithreaded design. Using a class that supports this pattern can allow you to:-
+ * (1) Perform time-consuming tasks, such as downloads and database operations, "in the background," without
+ * interrupting your application. (2) Execute multiple operations simultaneously, receiving notifications when each
+ * completes. (3) Wait for resources to become available without stopping ("hanging") your application. (4) Communicate
+ * with pending asynchronous operations using the familiar events-and-delegates model.
+ * 
+ * @see EventManager
+ * @see Event
+ *
+ */
+public class App {
+
+  public static final String PROP_FILE_NAME = "config.properties";
+
+  boolean interactiveMode = false;
+
+  /**
+   * Program entry point.
+   *
+   * @param args command line args
+   */
+  public static void main(String[] args) {
+    App app = new App();
+
+    app.setUp();
+    app.run();
+  }
+
+  /**
+   * App can run in interactive mode or not. Interactive mode == Allow user interaction with command line.
+   * Non-interactive is a quick sequential run through the available {@link EventManager} operations.
+   */
+  public void setUp() {
+    Properties prop = new Properties();
+
+    InputStream inputStream = App.class.getClassLoader().getResourceAsStream(PROP_FILE_NAME);
+
+    if (inputStream != null) {
+      try {
+        prop.load(inputStream);
+      } catch (IOException e) {
+        System.out.println(PROP_FILE_NAME + " was not found. Defaulting to non-interactive mode.");
+      }
+      String property = prop.getProperty("INTERACTIVE_MODE");
+      if (property.equalsIgnoreCase("YES")) {
+        interactiveMode = true;
+      }
+    }
+  }
+
+  /**
+   * Run program in either interactive mode or not.
+   */
+  public void run() {
+    if (interactiveMode) {
+      runInteractiveMode();
+    } else {
+      quickRun();
+    }
+  }
+
+  /**
+   * Run program in non-interactive mode.
+   */
+  public void quickRun() {
+    EventManager eventManager = new EventManager();
+
+    try {
+      // Create an Asynchronous event.
+      int aEventId = eventManager.createAsync(60);
+      System.out.println("Event [" + aEventId + "] has been created.");
+      eventManager.start(aEventId);
+      System.out.println("Event [" + aEventId + "] has been started.");
+
+      // Create a Synchronous event.
+      int sEventId = eventManager.create(60);
+      System.out.println("Event [" + sEventId + "] has been created.");
+      eventManager.start(sEventId);
+      System.out.println("Event [" + sEventId + "] has been started.");
+
+      eventManager.status(aEventId);
+      eventManager.status(sEventId);
+
+      eventManager.cancel(aEventId);
+      System.out.println("Event [" + aEventId + "] has been stopped.");
+      eventManager.cancel(sEventId);
+      System.out.println("Event [" + sEventId + "] has been stopped.");
+
```

Comment:
Can we change the output so that it tells whether we are talking about sync or async event. For example "Async event [xxx] has been created" and "Sync event [xxx] has been created"?